setup.py

```diff
@@ -1015,19 +1015,15 @@ def detect_simple_extensions(self):
         self.addext(Extension("_json", ["_json.c"]))
 
         # profiler (_lsprof is for cProfile.py)
-        self.add(Extension('_lsprof', ['_lsprof.c', 'rotatingtree.c']))
+        self.addext(Extension('_lsprof', ['_lsprof.c', 'rotatingtree.c']))
         # static Unicode character database
         self.addext(Extension('unicodedata', ['unicodedata.c']))
-        # _opcode module
-        self.add(Extension('_opcode', ['_opcode.c']))
+        self.addext(Extension('_opcode', ['_opcode.c']))
         # asyncio speedups
-        self.add(Extension("_asyncio", ["_asynciomodule.c"]))
-        # _queue module
-        self.add(Extension("_queue", ["_queuemodule.c"]))
-        # _statistics module
-        self.add(Extension("_statistics", ["_statisticsmodule.c"]))
-        # _typing module
-        self.add(Extension("_typing", ["_typingmodule.c"]))
+        self.addext(Extension("_asyncio", ["_asynciomodule.c"]))
+        self.addext(Extension("_queue", ["_queuemodule.c"]))
+        self.addext(Extension("_statistics", ["_statisticsmodule.c"]))
+        self.addext(Extension("_typing", ["_typingmodule.c"]))
```

Comment:
You removed some of the comments. Now it looks like all four C extensions are asyncio speedups.